src/main/java/org/haffson/adventofcode/days/day02/Day02.java

```diff
@@ -0,0 +1,177 @@
+package org.haffson.adventofcode.days.day02;
+
+import org.haffson.adventofcode.ProblemStatusEnum;
+import org.haffson.adventofcode.days.Days;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Scanner;
+
+
+
+/**
+ * Implementation for <i>Day 1: Chronal Calibration</i>.
+ */
+@Component
+public class Day02 implements Days {
+
+    private final String path = "/Users/jenni/dedica/AdventOfCode/Jenni/Day02/src/input.txt";
+    private List<String> rawData = getRawDataAsList(path);
+
+    /** The puzzle status {@code HashMap} */
+    private final HashMap<String, ProblemStatusEnum> problemStatus;
+
+    @Autowired
+    Day02() {
+        this.problemStatus = new HashMap<>();
+        this.problemStatus.put("1", ProblemStatusEnum.SOLVED);
+        this.problemStatus.put("2", ProblemStatusEnum.SOLVED);
+    }
+
+    @Override
+    public int getDay() {
+        return 2;
+    }
+
+    @Override
+    public HashMap<String, ProblemStatusEnum> getProblemStatus() {
+        return problemStatus;
+    }
+
+    @Override
+    public String firstPart() {
+        try {
+            return "Part 1 answer: " + getNumberPwd1();
+        } catch (FileNotFoundException e) {
+            return "Error in method 1";
+        }
+    }
+
+    @Override
+    public String secondPart() {
+        try {
+            return "Part 2 answer: " + getNumberPwd2();
+        } catch (FileNotFoundException e) {
+            return "Error in method 2";
+        }
+    }
+
+
+    /** Method to read raw data from file into list
+     *
+     * @return raw data as list
+     */
+    public List<String> getRawDataAsList(String path) {
+        List<String> rawData = new ArrayList<>();
+
+        try {
+            Scanner s = new Scanner(new File(path)).useDelimiter("\n");
+            while (s.hasNext()) {
+                rawData.add(s.next());
+            }
+            s.close();
+        } catch (FileNotFoundException e) {
+            System.out.println("File not found!");
+        }
+        return rawData;
+    }
+
+    /**
+     * Method getData() to get min and max number, letter and password from raw data
+     * First, create class Data to avoid having multiple return values
+     */
+    public static class Data {
+        int min;
+        int max;
+        String letter;
+        String pwd;
+
+        public Data(int min, int max, String letter, String pwd) {
+            this.min = min;
+            this.max = max;
+            this.letter = letter;
+            this.pwd = pwd;
+        }
+    }
+
+    public List<Data> getData(List<String> rawData) {
+
+        List<Data> dataArrayList = new ArrayList<>();
+
+        for (int i = 0; i < rawData.size(); i++) {
+
+            String line = rawData.get(i);
```

Comment:
This is perfectly fine as it is, but it can be replaced with an "enhanced" for loop:
```suggestion
        for (String line : rawData) {
```